src/Drupal/Driver/DrushDriver.php

```diff
@@ -189,11 +189,27 @@ public function clearStaticCaches() {
   }
 
   /**
+   * Decodes JSON output returned by Drush.
+   *
+   * It will clean up any junk that may have appeared before the JSON.
+   *
+   * @param string $output
+   *   The output from Drush.
+   * @return object
+   *   The decoded JSON object.
+   */
+  protected function decodeJsonOutput($output) {
+    // Remove anything before the first '{'.
+    $output = preg_replace('/^[^\{]*/', '', $output);
```

Comment:
@greg-1-anderson Your regex wizardry worked! On thinking through it again, it does finally make sense to me. With [your first regex](https://github.com/jhedstrom/DrupalDriver/pull/91/files/bc98ee0ef2795df76b28e6189361c0461e4c97c8#r66506455), the engine would probably start capturing immediately, and each time it met a `}` it would clear what it had previously captured, and start capturing again, until ultimately it met the end of the buffer and the last bit that it captured would be the match. Or some variation on that. Regexes are hard. :-)

I've updated the PR. I chose to do it in two separate lines for clarity, but I'd be happy to switch it to the one line version if you guys preferred! Thanks again!